package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-yola",
-  "version": "0.2.1",
+  "version": "0.2.1-dev.1",
```

Comment:
shouldnt here be <next_version>-dev.1?

Reply:
@AnastasiiaLisovska our flow of version bump is that we're open separate PR with new version proposal after changes been approved. So here I assume that there is no new version yet while my changes get approve or discard.